packages/fast-components-react-base/src/listbox/listbox.props.ts

```diff
@@ -0,0 +1,32 @@
+import * as React from "react";
+import {
+    ListboxClassNameContract,
+    ManagedClasses,
+} from "@microsoft/fast-components-class-name-contracts-base";
+
+export interface ListboxManagedClasses extends ManagedClasses<ListboxClassNameContract> {}
+export interface ListboxUnhandledProps extends React.HTMLAttributes<HTMLDivElement> {}
+export interface ListboxHandledProps extends ListboxManagedClasses {
+    /**
+     * The listbox children
+     */
+    children?: React.ReactNode;
+
+    /**
+     * The name of the prop on child components that is used as the data for type ahead focus
+     */
+    typeAheadPropertyKey?: string;
+
+    /**
+     * Whether this listbox supports multi-selection (default is 'false')
+     */
+    multiselectible?: boolean;
+
+    /**
+     * The aria-labelledby attribute to link the listbox to an existing
+     * element that provides it an accessible name
+     */
+    labelledBy?: string;
```

Comment:
Nowhere in here is a callback for when something has actually been selected.

Reply:
Revisit after I've refactored selection logic out of the parent Select into the Listbox itself.

Reply:
Refactored listbox to include selection